creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -763,30 +764,17 @@ class SnapbackSM {
            * If either secondary is in `unhealthyPeers` list, add it to `unhealthyReplicas` list
            */
           for (const secondary of secondaries) {
-            if (unhealthyPeers.has(secondary)) {
+            const userSecondarySyncMetrics = await SecondarySyncHealthTracker.computeUserSecondarySyncSuccessRates(
```

Comment:
better to do this call outside this for loop, with secondaries instead of one secondary at a time

Reply:
rest of this change looks good

Reply:
moved to outside loop